src/back/TicketManager/src/Core/TicketManager.Core.Contracts/Organizers/OrganizerDecideRequest.cs

```diff
@@ -0,0 +1,7 @@
+namespace TicketManager.Core.Contracts.Organizers;
+
+public class OrganizerDecideRequest
+{
+    public Guid OrganizerId { get; set; }
+    public bool IsAccepted { get; set; }
```

Comment:
Add validation here for `OrganizerDoesNotExist` and `OrganizerIsAlreadyVerified`.